aikiblog/templates/aikiblog/parts/training_list.html

```diff
@@ -15,7 +15,10 @@
             stażu
             {% endifequal %}
             <br> <a href="{% url "training-detail" entry.slug %}"> {{ entry.date|date:"d E Y" }} o {{ entry.date|date:"H:i" }} w {{ entry.place }}</a>
-        <div id="comments">Komentarze(0)&nbsp;|&nbsp;Dodaj komentarz</div></div>
+        <div id="comments"><a href="{% url "training-detail" entry.slug %}#comments">Komentarze({{ entry.trainingcomment_set.all.count }})</a>
+            {% if request.user.is_authenticated %}|&nbsp;<a href="{% url "training-detail" entry.slug %}#comments">Dodaj komentarz</a>
+            {% endif %}
+        </div></div>
         <div id="post_body">
 
         </div>
```

Comment:
W tej templatce jest mocno pomieszane z wcięciami. Trochę to ogarnę i popatrz sobie.
W templatkach mamy jako wcięcia taby, w plikach py mamy 4 spacje na jedno wcięcie.